android/build.gradle

```diff
@@ -58,7 +58,7 @@ ext {
     clientmanagerLibVersionCode = 1
     clientmanagerLibVersionName = "\"1.0.0\""
 
-    serverBaseURL = "\"https://api-internal.qa-base.mosip.net\""
+    serverBaseURL = "\"https://api-internal.qa-core.mosip.net\""
```

Comment:
@ckm007 @Prafulrakhade  can we take this as build argument?

Reply:
Let's create a jiara ticket for this.